back/spec/acceptance/users_spec.rb

```diff
@@ -930,6 +930,19 @@
             expect(json_response.dig(:data, :attributes, :custom_field_values)).not_to include(cf.key.to_sym)
             expect(@user.custom_field_values[cf.key]).to eq(some_value)
           end
+
+          # To allow for custom fields to be required or not depending on the action
+          example 'Allow update if custom fields are changed but required fields are not present' do
```

Comment:
Do we want to include this in the API docs? If not, then we could add `document: false`.